config/v1/types_infrastructure.go

```diff
@@ -457,6 +462,32 @@ type AWSResourceTag struct {
 	Value string `json:"value"`
 }
 
+type ClusterDNSConfig struct {
+	// APIServerDNSConfig contains DNS configuration for the API Server.
+	// +optional
+	APIServerDNSConfig []DNSConfig `json:"apiServerDNSConfig,omitempty"`
```

Comment:
For this and the others, this is a slice, what does it mean for there to be more than one entry? Can there be an infinite number of entries within here?

Reply:
We expect one LB IP address for each availability zone. If dualstack is supported that number would be doubled; one per address family for each LB in availability zone.

Reply:
ok, so perhaps we can set a sensible limit on this and the other slices, maybe a limit of 256 would be sufficient for most cases?